MediaBrowser.Controller/SyncPlay/GroupInfo.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Session;
+
+namespace MediaBrowser.Controller.SyncPlay
+{
+    /// <summary>
+    /// Class GroupInfo.
+    /// </summary>
+    /// <remarks>
+    /// Class is not thread-safe, external locking is required when accessing methods.
+    /// </remarks>
+    public class GroupInfo
+    {
+        /// <summary>
+        /// Default ping value used for sessions.
+        /// </summary>
+        public readonly long DefaulPing = 500;
+        /// <summary>
+        /// Gets or sets the group identifier.
+        /// </summary>
+        /// <value>The group identifier.</value>
+        public readonly Guid GroupId = Guid.NewGuid();
```

Comment:
Fields should not be public

Reply:
Just found out that auto-properties exist, exactly what I needed.